main.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
And then if you do that, I think we might be able to just go ahead and delete all of `vendor/fbtransform`.